stories/form/form-masked-input/form-masked-input.mdx

```diff
@@ -17,6 +17,18 @@ import { Form } from "@lifesg/react-design-system/form";
 
 <Canvas of={FormMaskedInputStories.Default} />
 
+<Heading3>Loading display</Heading3>
+
+In some cases where the unmasking requires an asynchronous process, you might want to display
+a loading indicator for the users. As such this component comes with a loading display. Here is
+an example of how you can set things up.
+
+> Note: This only applies if the component is in `readOnly` mode.
+> You will also have to handle the masking and unmasking manually to ensure the value changes
```

Comment:
can soften the wording here from **will** to **may** imo. this is not necessary for simple masking cases like uinfin with a fixed `maskedRange` as the initial masking logic can still apply